### DISCUSSION TITLE
Direction Accuracy and Loss

### DISCUSSION THREAD

**Discussion by minion50000** [mltrev23/snp500_stock_prediction#10] [Ideas]
I think it would be better if we try the model training based on the direction loss
prediction_dir = np.sign(prediction[:-1] - prediction[1:])
truth_dir = np.sign(truth[:-1] - truth[1:])
dir_acc = np.mean(prediction_dir == truth_dir)
What do you think? 😉

**Comment by mltrev23** [comment]
Great
You hit the nail on the head! 💯 🙌 
Actually, it could improve the model's performance, imo.
👍

**Comment by mltrev23** [comment]
could you please create pr or will I?

**Reply by minion50000** [reply]
okay, i will
